### ISSUE TITLE
unable to serve documentation locally

### ISSUE THREAD

**Issue by nicodebo** [cookiecutter/cookiecutter#1062]
* Cookiecutter version: 1.6.0
* Template project url: none
* Python version: 3.6
* Operating System: archlinux

### Description:

As stated in CONTRIBUTING.md, in the write documentation section, to review change I need to run the following commands:

```bash
pip install -r docs/requirements.txt
make servedocs
```

The `make servedocs` command return the following error: 
```
Could not import extension docs.ccext
```

I expect no error from the `make command` and be able to review my local changes to the documentation.

### What I've run:

```
❯ pip install -r docs/requirements.txt
Collecting watchdog==0.8.3 (from -r docs/requirements.txt (line 1))
Collecting sphinx-rtd-theme==0.1.9 (from -r docs/requirements.txt (line 2))
  Using cached sphinx_rtd_theme-0.1.9-py3-none-any.whl
Collecting pathtools>=0.1.1 (from watchdog==0.8.3->-r docs/requirements.txt (line 1))
Collecting PyYAML>=3.10 (from watchdog==0.8.3->-r docs/requirements.txt (line 1))
Collecting argh>=0.24.1 (from watchdog==0.8.3->-r docs/requirements.txt (line 1))
  Using cached argh-0.26.2-py2.py3-none-any.whl
Collecting sphinx>=1.3 (from sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached Sphinx-1.7.2-py2.py3-none-any.whl
Collecting Jinja2>=2.3 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached Jinja2-2.10-py2.py3-none-any.whl
Requirement already satisfied: setuptools in /home/debz/.local/share/virtualenvs/pull_cookie/lib/python3.6/site-packages (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
Collecting packaging (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached packaging-17.1-py2.py3-none-any.whl
Collecting requests>=2.0.0 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached requests-2.18.4-py2.py3-none-any.whl
Collecting alabaster<0.8,>=0.7 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached alabaster-0.7.10-py2.py3-none-any.whl
Collecting snowballstemmer>=1.1 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached snowballstemmer-1.2.1-py2.py3-none-any.whl
Collecting Pygments>=2.0 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached Pygments-2.2.0-py2.py3-none-any.whl
Collecting six>=1.5 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached six-1.11.0-py2.py3-none-any.whl
Collecting babel!=2.0,>=1.3 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached Babel-2.5.3-py2.py3-none-any.whl
Collecting imagesize (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached imagesize-1.0.0-py2.py3-none-any.whl
Collecting sphinxcontrib-websupport (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached sphinxcontrib_websupport-1.0.1-py2.py3-none-any.whl
Collecting docutils>=0.11 (from sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached docutils-0.14-py3-none-any.whl
Collecting MarkupSafe>=0.23 (from Jinja2>=2.3->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
Collecting pyparsing>=2.0.2 (from packaging->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached pyparsing-2.2.0-py2.py3-none-any.whl
Collecting idna<2.7,>=2.5 (from requests>=2.0.0->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached idna-2.6-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.0.0->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached chardet-3.0.4-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests>=2.0.0->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached certifi-2018.1.18-py2.py3-none-any.whl
Collecting urllib3<1.23,>=1.21.1 (from requests>=2.0.0->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached urllib3-1.22-py2.py3-none-any.whl
Collecting pytz>=0a (from babel!=2.0,>=1.3->sphinx>=1.3->sphinx-rtd-theme==0.1.9->-r docs/requirements.txt (line 2))
  Using cached pytz-2018.3-py2.py3-none-any.whl
Installing collected packages: pathtools, PyYAML, argh, watchdog, MarkupSafe, Jinja2, pyparsing, six, packaging, idna, chardet, certifi, urllib3, requests, alabaster, snowballstemmer, Pygments, pytz, babel, imagesize, sphinxcontrib-websupport, docutils, sphinx, sphinx-rtd-theme
Successfully installed Jinja2-2.10 MarkupSafe-1.0 PyYAML-3.12 Pygments-2.2.0 alabaster-0.7.10 argh-0.26.2 babel-2.5.3 certifi-2018.1.18 chardet-3.0.4 docutils-0.14 idna-2.6 imagesize-1.0.0 packaging-17.1 pathtools-0.1.2 pyparsing-2.2.0 pytz-2018.3 requests-2.18.4 six-1.11.0 snowballstemmer-1.2.1 sphinx-1.7.2 sphinx-rtd-theme-0.1.9 sphinxcontrib-websupport-1.0.1 urllib3-1.22 watchdog-0.8.3

❯ make servedocs 
+ docs
Creating file docs/cookiecutter.rst.
Creating file docs/modules.rst.
make[1]: Entering directory '/tmp/cookiecutter/docs'
rm -rf _build/*
make[1]: Leaving directory '/tmp/cookiecutter/docs'
make[1]: Entering directory '/tmp/cookiecutter/docs'
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.7.2

Extension error:
Could not import extension docs.ccext (exception: No module named 'click')
make[1]: *** [Makefile:53: html] Error 2
make[1]: Leaving directory '/tmp/cookiecutter/docs'
make: *** [Makefile:67: docs] Error 2
```

**Comment by lumega** [comment]
I will be working in this issue

**Comment by lumega** [comment]
please check and apply this PR
if this commit is applied, it will:
*  add missing dependencies
https://github.com/audreyr/cookiecutter/pull/1126

**Comment by insspb** [comment]
Fixed in #1216
